pyveda/frameworks/batch_generator.py

```diff
@@ -12,13 +14,17 @@ class BaseGenerator():
     Rescale: Boolean. Flag to indicate if data returned from the generator should be rescaled between 0 and 1.
     '''
 
-    def __init__(self, cache, batch_size=32, shuffle=True, rescale=False):
+    def __init__(self, cache, batch_size=32, shuffle=True, rescale=False,
+                    flip_horizontal=False, flip_vertical=False):
         self.cache = cache
         self.batch_size = batch_size
         self.index = 0
         self.shuffle = shuffle
         self.rescale = rescale
         self.on_epoch_end()
+        self.flip_h = flip_horizontal
+        self.flip_v = flip_vertical
+        self._applied_augs(self.flip_h, self.flip_v)
```

Comment:
how come this function is called here? looks like it doesn't change any state, just returns something that's intended to be used for something else, but the function return not handled

Reply:
yeah since `self._applied_augs` isn't really don't anything there I've removed it.